testfiles/TF06/06.2-pass-1.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>Decorative Image - role equals presentation (TC06.2-all-pass-1)</title>
+    <title>Decorative Image - role equals presentation (TC06.2-pass-1)</title>
```

Comment:
reference the spreadsheet for filename guidance: 

- passing cases are "#-pass"; 
- failing cases are "all-fail"